app/assets/javascripts/thredded/core/on_page_load.es6

```diff
@@ -0,0 +1,40 @@
+(function () {
```

Comment:
just a couple of queries:

* why is `($)` not a parameter of the anon function (isn't that part of the point of the jQuery indirection?)
* is there a reason why a function and not an es6 fat arrow? (feels like we should be preferring es6 fat arrows in general unless there's a [reason not to use them](https://rainsoft.io/when-not-to-use-arrow-functions-in-javascript/)